Caboodle/FileSystem/FileSystem.ios.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.IO;
+using System.Threading.Tasks;
+using Foundation;
+
+namespace Microsoft.Caboodle
+{
+    public static partial class FileSystem
+    {
+        private static string cache;
+        private static string appData;
+        private static string userData;
+
+        public static string CacheDirectory
+            => cache ?? (cache = GetDirectory(NSSearchPathDirectory.CachesDirectory));
+
+        public static string AppDataDirectory
```

Comment:
I guess should this be library or this one...? https://developer.apple.com/library/content/documentation/FileManagement/Conceptual/FileSystemProgrammingGuide/FileSystemOverview/FileSystemOverview.html  

This is the top-level directory for any files that are not user data files. You typically put files in one of several standard subdirectories. iOS apps commonly use the Application Support and Caches subdirectories; however, you can create custom subdirectories.

Use the Library subdirectories for any files you don’t want exposed to the user. Your app should not use these directories for user data files.

On Android we return the root.... aka Library